static/app/gettingStartedDocs/capacitor/capacitor.tsx

```diff
@@ -1,229 +1,409 @@
-import {Fragment} from 'react';
-
 import {Layout, LayoutProps} from 'sentry/components/onboarding/gettingStartedDoc/layout';
 import {ModuleProps} from 'sentry/components/onboarding/gettingStartedDoc/sdkDocumentation';
 import {StepType} from 'sentry/components/onboarding/gettingStartedDoc/step';
+import {PlatformOption} from 'sentry/components/onboarding/gettingStartedDoc/types';
+import {getUploadSourceMapsStep} from 'sentry/components/onboarding/gettingStartedDoc/utils';
+import {useUrlPlatformOptions} from 'sentry/components/onboarding/platformOptionsControl';
+import {ProductSolution} from 'sentry/components/onboarding/productSelection';
 import {t, tct} from 'sentry/locale';
+import type {Organization, PlatformKey} from 'sentry/types';
+
+export enum SiblingOption {
+  ANGULARV10 = 'angularV10',
+  ANGULARV12 = 'angularV12',
+  REACT = 'react',
+  VUE3 = 'vue3',
+  VUE2 = 'vue2',
+}
+
+type PlaformOptionKey = 'siblinOption';
+
+type StepProps = {
+  errorHandlerProviders: string;
+  sentryInitContent: string;
+  siblingOption: SiblingOption;
+  newOrg?: boolean;
+  organization?: Organization;
+  platformKey?: PlatformKey;
+  projectId?: string;
+};
+
+const IsAngular = (siblingOption: SiblingOption): boolean =>
+  siblingOption === SiblingOption.ANGULARV10 ||
+  siblingOption === SiblingOption.ANGULARV12;
+
+const IsVue = (siblingOption: SiblingOption): boolean =>
+  siblingOption === SiblingOption.VUE2 || siblingOption === SiblingOption.VUE3;
 
 // Configuration Start
-export const steps = ({
-  dsn,
-}: Partial<Pick<ModuleProps, 'dsn'>> = {}): LayoutProps['steps'] => [
-  {
-    type: StepType.INSTALL,
-    description: t(
-      'Install the Sentry Capacitor SDK alongside the sibling Sentry Angular SDK:'
-    ),
-    configurations: [
+const platformOptions: Record<PlaformOptionKey, PlatformOption> = {
+  siblinOption: {
```

Comment:
ATM Angular 12+ is the selected default.
Does this make sense or should we rather select React?

Reply:
I see no problem in selecting Angular, but if needed I can change it to React.

Reply:
k 👍 was just wondering which of those is more common when using Capacitor.